src/server/api/routes/router.js

```diff
@@ -8,13 +8,49 @@ const userRouter = require("./user.router");
 const authRouter = require("./auth.router");
 const refreshTokenRouter = require("./refresh.token.router");
 const mailRouter = require("./mail.router");
+
+const vesselReportRouter = require("./vessel-reports.router");
+
 const portRouter = require("./port.router");
 const voyagesRouter = require("./voyages.router");
 const suggestedRoutesRouter = require("./suggested-routes.router");
 const organizationRouter = require("./organizations.router");
 const vesselRouter = require("./vessel.router");
+
 const favoriteVesselsRouter = require("./favorite-vessels.router");
 
+// /api/users
+router.use("/users", userRouter);
+
+// /api/auth
+router.use("/auth", authRouter);
+
+// /api/modules
+router.use("/modules", modulesRouter);
+
+router.use("/mail", mailRouter);
+
+router.use("/token", refreshTokenRouter);
+
+// /api/vessel-reports
+router.use("/vessel-reports", vesselReportRouter);
+
+router.use("/ports", portRouter);
+
+// /api/suggested-routes
+router.use("/suggested-routes", suggestedRoutesRouter);
+
+// /api/organizations
+router.use("/organizations", organizationRouter);
+router.use("/vessels", vesselRouter);
+// /api/organizations/organization_id/vessels
+
+// /api/voyages
+router.use("/voyages", voyagesRouter);
+
+// /api/favorite_vessels/
+router.use("/favorite-vessels", favoriteVesselsRouter);
+
```

Comment:
You've not resolved this correctly. All of the `router.use` statements above should look similar to this:

```
router.use(`${process.env.API_PATH}/users`, userRouter);
```
(of course replace `users` and `userRouter`)

Also make sure you have API_PATH set in `.env` to test it.

Reply:
I think you killed the API docs? There should be a route for them here in this file. You probably just rejected all of the incoming changes in the merge. Not good 😢